internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -2548,6 +2554,148 @@ components:
         license:
           type: string
 
+    Disk:
+      type: object
+      required:
+        - partitions
+      properties:
+        type:
+          type: string
+          enum:
+            - gpt
+            - dos
+          description: |
+            Type of the partition table
+        minsize:
+          x-go-type: uint64
+          example: 2147483648
```

Comment:
IIRC we support strings like "20 GiB" here, would it make sense to use them in the example(s)?

Reply:
I see that other sizes are also just uints in the cloud API.  I think we just don't support strings here and the IB middleware probably takes care of the string parsing?

Reply:
We could do this either way, either accept an int or string, or both. Will require some manual parsing if it's string though.

Reply:
Let me try to do it in the spec and see what the code looks like.

Reply:
Answer: Not great.

The following:
```yaml
    minsize:
      oneOf:
        - name: minsize
          x-go-type: uint64
          example: 2147483648
          description: 'size in bytes'
        - name: minsize
          type: string
          description: 'size with data units'
```
works, but in Go it just creates a `Minsize` type alias to `interface{}`.  It's not impossible to work with, but it needs some fiddling.

Reply:
Used that `oneOf` and handled parsing.  PTAL.